config/server.json

```diff
@@ -11,7 +11,8 @@
     "roleattr": "businessCategory",
     "defaultroleattr": "employeeType",
     "groupclassattr": "groupOfNames",
-    "pubkeysattr": "sshPublicKey"
+    "pubkeysattr": "sshPublicKey",
+    "roletimeoutattr": "description"
```

Comment:
placeholder?

Reply:
There aren't many attributes that the group subschema allows and I think description is the most fitting.

Reply:
ack

Reply:
As this file is merely an example/placeholder anyway, perhaps it would be better to omit this somewhat confusing item.  I imagine most users needing this will be able to use a custom schema / better attribute name.

Reply:
I changed this to a dummy value so that it's still in the configuration as a way of documenting this feature but better explains its purpose.